### PR TITLE
[Feature] Profile and applications page update

### LINKED ISSUE #6937: 📝 Update copy on the "Profile and applications" page
# ✨ Details ([Figma file](https://www.figma.com/file/jHsXzqMfBtomLa1Gom94Y7/Candidates---Profile-and-applications?type=design&node-id=1097-34868))

This update changes a few minor copy elements on the page for both consistency and clarity.

# 🗃️ Requirements

```[tasklist]
## 📝 Page-wide copy
- [x] `Résumé and recruitments` >> `Résumé and recruitment`
- [x] `Community participation`'s icon should be changed from 2 users to 3
- [x] A `Browse jobs` link has been added inline with the `Track your applications` title
```

```[tasklist]
## ⚠️ Mid-IAP application alert
- [x] Update the alert content
```

```
Are you looking for your application to the IT Apprenticeship Program for Indigenous Peoples?

When you log into your account, you’ll start on this page from now on. 

The apprenticeship program is a part of a larger digital talent initiative in the Government of Canada that aspires to recruit top digital talent like yourself. You can use the “Track your applications” section on this page to review the status of your application to the IT Apprenticeship Program for Indigenous Peoples.

This profile provides access to other recruitment opportunities beyond the apprenticeship program if you choose to apply. If you’d prefer to stick to the apprenticeship program exclusively, that’s okay too!
```

```[tasklist]
### ⚠️ Post-IAP application alert
- [x] Update the alert content
```

```
You've now landed on your Digital Talent profile page.

When you log into your account, you’ll start on this page from now on. 

The apprenticeship program is a part of a larger digital talent initiative in the Government of Canada that aspires to recruit top digital talent like yourself. You can use the “Track your applications” section on this page to review the status of your application to the IT Apprenticeship Program for Indigenous Peoples.

This profile provides access to other recruitment opportunities beyond the apprenticeship program if you choose to apply. If you’d prefer to stick to the apprenticeship program exclusively, that’s okay too!
```

### PR DESCRIPTION
**Pull request by petertgiles**
🤖 Resolves #6937 

## 👋 Introduction

This branch has a grab-bag of update to the Profile and applications page and subpages.

## 🕵️ Details

I suggest reviewing this commit-by-commit since the "[rename page](https://github.com/GCTC-NTGC/gc-digital-talent/commit/4405ae09dc90823b026b37ed5f7f802eb8b4597d)" commit has so many files touched in it.

## 🧪 Testing

1. Rebuild the app
2. Check that the ACs are met
    - Visit `/en/applicant/dashboard?fromIapSuccess=true&fromIapDraft=true` to pop the notifications 
4. Ensure the rename didn't break anything

## 📸 Screenshot
image

### CODE REVIEW

File: apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx [hunk 1]
```diff
@@ -206,36 +214,45 @@ const DashboardHeading = ({ user }: DashboardHeadingProps) => {
...
             })}
           </p>
-          <p>
+          <p data-h2-margin-bottom="base(x0.5)">
             {intl.formatMessage(
               {
                 defaultMessage:
-                  "When you log into your account, you’ll start on this page from now on. You can use the <strong><a>Track your applications</a></strong> section on this page to review your Indigenous apprenticeship application and track your status in the program.",
-                id: "RzNVWf",
+                  'The apprenticeship program is a part of a larger digital talent initiative in the Government of Canada that aspires to recruit top digital talent like yourself. You can use the "<a>Track your applications</a>" section on this page to review the status of your application to the IT Apprenticeship Program for Indigenous Peoples.',
```
Comment (vd1992) [inline apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx:236]: Just a reminder I think the `" "` in the translations will need that special character `« »`

**Comment by vd1992** [comment]
Is there already an issue to resolve this, it is two-fold. 

"You are appearing in results" for a draft application.
And
Missing a continue draft on the card, you have to head to the poster and then continue from there.

gif

**Comment by petertgiles** [comment]
> Is there already an issue to resolve this, it is two-fold.

Yeah, I noticed this, too.  You can see it [on main in Chromatic](https://www.chromatic.com/component?appId=61e099a1bb1465003a73d3ce&csfId=components-track-applications-card&buildNumber=5143&k=649336d7649775225005e079-1200-interactive-true&h=7&b=-1), as well.  There are quite a few states with issues.
image

@brindasasi Are you noticing this in #6738?  Is it within scope for you to fix these?

**Comment by brindasasi** [comment]
@petertgiles yes. I Do.
My PR should resolve of this.

File: apps/web/src/lang/fr.json
[the remark this answers is not in the document]
Comment (petertgiles) [reply apps/web/src/lang/fr.json:1]: Thanks, good catch!
Ref: [fix mismatched description in fr file](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7029/commits/1d1d04d7484ee7864709caa21fa87d5b53547b30)
(The removals in this commit are from pending translations that haven't come back yet.)

File: apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx (hunk 1 above)
Comment (petertgiles) [reply apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx:236]: Good reminder!  I'll resolve this component when the translation comes back and I've confirmed.

**Review by vd1992 — APPROVED** [review]
Comments above addressed ✅ 
Issue noted and to be resolved elsewhere ✅ 
Chromatic ✅ 
Hopping around working ✅ 

💯

File: apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx (hunk 1 above)
Comment (petertgiles) [reply apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx:236]: Yep, I had to manually fix that.  :wink:

**Pull request merged**